install/updates/10-config.update

```diff
@@ -15,10 +15,8 @@ only: nsslapd-pluginPrecedence: 60
 
 # Set limits to suite better IPA deployment sizes, defaults are too
 # conservative
-dn: cn=config
-default: nsslapd-sizelimit:100000
-
-dn: cn=config,cn=ldbm database,cn=plugins,cn=config
+dn: cn=default instance config,cn=chaining database,cn=plugins,cn=config
+replace: nsslapd-sizelimit:2000::100000
```

Comment:
It's not, https://access.redhat.com/documentation/en-us/red_hat_directory_server/10/html/administration_guide/finding_directory_entries says it should be in cn=config. I think what I did was look in dse.ldif and found rather than looking in the docs.